grails-plugin-gsp/src/main/groovy/org/grails/plugins/web/taglib/FormTagLib.groovy

```diff
@@ -85,7 +90,7 @@ class FormTagLib implements ApplicationContextAware, InitializingBean, TagLibrar
                 springSecurityCsrfTokenClass =
                         Class.forName("org.springframework.security.web.csrf.CsrfToken")
             }
-        } catch (ClassNotFoundException ignore) {}
+        } catch (ClassNotFoundException | BeansException ignore) {}
```

Comment:
This works.  My only question is you want to ignore all bean exceptions rather than just `NoSuchBeanDefinitionException`?

Reply:
@bkoehm because there could be a situation where you have no bean and also a situation where you have 2 beans the you should only have 1. this catches both of them.  There could very well be a situation where somebody mistakenly has a duplicate registration. No need to fail because of that and both of those exception types extend BeansException.